### ISSUE TITLE
DF User Logger: italian localization is not working

### ISSUE THREAD

**Issue by smoothingplane** [flamewave000/dragonflagon-fvtt#283]
{ "lang": "it", "name": "Italiano", "path": "lang/it.json" } need to be changed in { "lang": "it", "name": "Italian", "path": "lang/it.json" },

**Comment by flamewave000** [comment]
You likely need to reset the messages. The "name" is just a label for users to see, the language code `it` is the thing Foundry cares about.

Open the module settings and click the "Modifica i Messaggi" button. This will open a menu where you can click a large red button that says "Ripristina Tutti i Messaggi"

**Comment by smoothingplane** [comment]
I tried to follow the method you recommended but everything remained in English (including settings).
I uninstalled and reinstalled the module and it continued to stay in English.
I tried to set Spanish as the language and the language changed.
![image](https://user-images.githubusercontent.com/43862671/148139343-771ea4f8-1774-4d82-87a9-5f7efe11e818.png)

![image](https://user-images.githubusercontent.com/43862671/148139178-b710f458-2109-4e68-bbb3-bad8cbb49b12.png)
